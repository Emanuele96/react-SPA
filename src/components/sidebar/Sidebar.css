.sidebar > h3 {
  font-size: 1.7rem;
  padding-bottom: 20px;
}

.sidebar {
  display: inline-block;
  border: rgb(4, 0, 255) dashed 1px;
  width:100%;

}