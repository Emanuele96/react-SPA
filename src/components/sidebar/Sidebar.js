import React, { Component } from 'react';
import './Sidebar.css';
import Soundform from '../soundform/Soundform';

class Sidebar extends Component {
  constructor(props) {
    super(props);
    this.state = {
      categorys: []
    };
  }

  render() {
    return (
      <div className='Sidebar'>
        <h3>This is Sidebar {this.props.name}!</h3>
        <Soundform/>
      </div>
    );
  }
}

export default Sidebar;
