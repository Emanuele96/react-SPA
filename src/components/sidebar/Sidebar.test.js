import React from 'react';
import renderer from 'react-test-renderer';

import Sidebar from './Sidebar';



it('renders correctly ', () => {
    const tree = renderer.create(<Sidebar 
        sendSoundCategory={()=> 0}
        sendTextCategory={()=> 0}
        sendPictureCategory={()=> 0}
        
        />).toJSON();
    expect(tree).toMatchSnapshot();
});



