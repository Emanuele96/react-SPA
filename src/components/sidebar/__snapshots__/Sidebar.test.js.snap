// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly  1`] = `
<div
  className="sidebar"
>
  <h3>
    Kategorier
  </h3>
  <div
    className="choice-form"
  >
    <h3>
      Sound
    </h3>
    <form>
      <label>
        <input
          checked={true}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Nature"
        />
        Nature
      </label>
      <label>
        <input
          checked={false}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Piano"
        />
        Piano
      </label>
      <label>
        <input
          checked={false}
          name="category"
          onChange={[Function]}
          type="radio"
          value="City Life"
        />
        City Life
      </label>
    </form>
  </div>
  <div
    className="choice-form"
  >
    <h3>
      Tekst
    </h3>
    <form>
      <label>
        <input
          checked={true}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Emosjoner"
        />
        Emosjoner
      </label>
      <label>
        <input
          checked={false}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Farger"
        />
        Farger
      </label>
      <label>
        <input
          checked={false}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Dyr"
        />
        Dyr
      </label>
    </form>
  </div>
  <div
    className="choice-form"
  >
    <h3>
      Bilder
    </h3>
    <form>
      <label>
        <input
          checked={true}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Animals"
        />
        Animals
      </label>
      <label>
        <input
          checked={false}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Cats"
        />
        Cats
      </label>
      <label>
        <input
          checked={false}
          name="category"
          onChange={[Function]}
          type="radio"
          value="Rats"
        />
        Rats
      </label>
    </form>
  </div>
</div>
`;
